spec/models/feature_flags_spec.rb

```diff
@@ -25,4 +25,22 @@
       end
     end
   end
+
+  describe ".enterprise_fee_summary_enabled?" do
```

Comment:
the convention of the `.` prefix is used to note this is a class method while `#` for instance ones. As this case is the latter, I would stick to it and use `#`. It will be confusing otherwise.

Reply:
Agh, yes thanks! I copied that from the other example group - fixed both already.